mne/io/kit/constants.py

```diff
@@ -161,7 +161,9 @@
 KIT.SYSTEM_UMD_2014_07 = 52  # UMD update to 16 bit ADC, July 4, 2014 -
 KIT.SYSTEM_UMD_2014_12 = 53  # UMD December 4, 2014 -
 KIT.SYSTEM_UMD_2019_09 = 54  # UMD September 3, 2019 -
-KIT.SYSTEM_YOKOGAWA = 1001  # no idea
+KIT.SYSTEM_YOKOGAWA_1001 = 1001  # Kanazawa (until 2017)
+KIT.SYSTEM_YOKOGAWA_10020 = 10020  # Kanazawa (since 2018)
+KIT.SYSTEM_YOKOGAWA_10021 = 10021  # Kanazawa (since August 2020)
```

Comment:
This do not follow the standard naming scheme. These should be:
```suggestion
KIT.SYSTEM_YOKOGAWA_2017_01 = 1001  # Kanazawa (until 2017)
KIT.SYSTEM_YOKOGAWA_2018_01 = 10020  # Kanazawa (since 2018)
KIT.SYSTEM_YOKOGAWA_2020_08 = 10021  # Kanazawa (since August 2020)
```